taxi/urls.py

```diff
@@ -42,10 +46,32 @@
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "cars/<int:pk>/driver_update/",
+        CarDriverUpdateView.as_view(),
+        name="car-driver-update",
     ),
+    path("drivers/",
+         DriverListView.as_view(),
+         name="driver-list"),
+
+    path("drivers/<int:pk>/",
+         DriverDetailView.as_view(),
+         name="driver-detail"),
+
+    path("drivers/create/",
+         DriverCreateView.as_view(),
+         name="drive-create"),
```

Comment:
The name for the path 'drivers/create/' is set as 'drive-create'. This seems to be a typo. Consider changing it to 'driver-create' to maintain consistency with other driver-related URL names.